content/en/golden_signals.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Golden Signals
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+[Golden signals](https://sre.google/sre-book/monitoring-distributed-systems/#xref_monitoring_golden-signals) are metrics that help us understand how [services](https://glossary.cncf.io/service/) are experienced by users. Today, it is the gold standard for monitoring application performance and focuses on four specific metrics: Latency, traffic, errors, and saturation. The first three metrics relate to requests. A request is a message sent from one service to another. Services do that to collaborate and work as a [distributed app](https://glossary.cncf.io/distributed_apps/).
+
+*Latency* is the time it takes to service a request. *Traffic* measures the request frequency placed on the application. *Errors* measure the request failure rate. And *Saturation* measures the capacity of app resources, allowing users to address bottlenecks before they become a problem.
+ 
+## Problem it addresses
+Without metrics, engineers have nothing to deduce why applications are behaving a certain way. While earlier monitoring tools were designed to monitor machines (not applications/services), modern monitoring systems gather a huge variety of parameters. They collect so many metrics that it's easy to lose focus on what is relevant for the user. Which of all these metrics should you focus on to ensure your customers have a great user experience?
```

Comment:
```suggestion
Finding out why an application behaves a certain way, especially when it doesn't behave like expected, can be a hard task for engineers. While earlier monitoring tools were more focused on monitoring machines (not applications/services), modern monitoring systems gather a huge variety of metrics. It's easy to lose focus on what is relevant for the user experience. Which of all these metrics should you focus on to ensure your customers have a great user experience?
```

Rephrased the first sentence because the statement was not true. Metrics are not the only way to find out why an application behaves a certain way. There a certain other ways to achieve that. For example logging, IDE assisted live debugging even code analysis.